aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -124,7 +125,7 @@ def __init__(self):
         self._cache: Optional[LRUDict] = None
         self.hash_function = None
 
-    def configure(self, config: IdempotencyConfig) -> None:
+    def configure(self, config: IdempotencyConfig, function_name: str) -> None:
```

Comment:
@heitorlessa - should this be optional? or required?

Reply:
Optional. Only because it's a public method and we have no way of knowing if someone took a dependency on it via tests or something - if they did, we will break them and take another release to fix it